setup.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+
+# Installing poetry
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+# Add poetry to PATH in script
+export POETRY_ROOT=\"$HOME//.poetry/\"
+export PATH=\"$POETRY_ROOT/bin:$PATH\"
+
+#Checking if poetry is already installed
+poetry --version 
```

Comment:
Here, it doesn't make much sense to test this _after_ you've just installed poetry - it's useful during writing the script for verifying the above, but in practice this step would be useful before installing poetry to determine whether we need to.